Cargo.lock

```diff
@@ -220,7 +220,7 @@ dependencies = [
 
 [[package]]
 name = "apollo-federation"
-version = "0.0.11"
+version = "1.46.0"
```

Comment:
@SimonSapin The fact that this version doesn't (and will likely never) exist on Crates.io shouldn't be a problem, right?

Reply:
No, I don’t expect it to matter.